src/io/ExportVTKXML.cpp

```diff
@@ -101,22 +101,22 @@ void ExportVTKXML::writeMasterFile(
 
   // write scalar data names
   outMasterFile << "      <PPointData Scalars=\"";
-  for (size_t i = 0; i < _scalarDataNames.size(); ++i) {
-    outMasterFile << _scalarDataNames[i] << ' ';
+  for (const auto &_scalarDataName : _scalarDataNames) {
+    outMasterFile << _scalarDataName << ' ';
```

Comment:
Maybe just `scalarDataName`, since it is not a member variable.

Reply:
Ah right, `clang-tidy` has chosen the names automatically. I will fix it, good catch.